graph/adjacency/ThingAdjacency.java

```diff
@@ -144,5 +176,56 @@
 
     }
 
-
+    EdgeDirected asSortable(ThingEdge edge);
+
+
+    abstract class EdgeDirected implements Comparable<EdgeDirected> {
+
```

Comment:
this is a new construct: we wrap an Edge which is undirected but therefore not sortable -- if it is an `In` adjacency, it may be sortable but in an `Out`. In other words, we can only sort in the direction of the adjacency. This wrapper enables us to return sorted edge iterators